src/Native/Unix/System.Native/pal_tcpstate.c

```diff
@@ -66,6 +66,9 @@ int32_t SystemNative_MapTcpState(int32_t tcpState)
             return TcpState_Closing;
         default:
             return TcpState_Unknown;
+#else
+        default:
+            return TcpState_Unknown;
```

Comment:
This default case now exists in the if, the else if, and the else.  Why not just move it to below the endif, remove it from the if and else if, and not have an else at all?